### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by sojanpr**
Removing some dead code

@Sarkar

### CODE REVIEW

File: Branch-SDK/src/io/branch/referral/SystemObserver.java [hunk 1]
```diff
@@ -374,31 +302,10 @@ public String getOS() {
...
-     * <b>generic</b>/vbox86p/vbox86p:5.0/LRX21M/buildbot12160004:userdebug/test-keys</pre>
-     *
-     * @return <p>A {@link Boolean} value indicating whether the device upon which the app is being run is
-     * a simulated platform, i.e. an emulator. Or a real, hardware device.</p>
-     * <ul>
-     * <li><i>true</i> - the app is running on an emulator</li>
-     * <li><i>false</i> - the app is running on a physical device (or a badly configured AVD)</li>
-     * </ul>
-     */
-    public boolean isSimulator() {
```
Comment (Sarkar) [inline Branch-SDK/src/io/branch/referral/SystemObserver.java]: I dont think we want to remove this

Comment (sojanpr) [inline Branch-SDK/src/io/branch/referral/SystemObserver.java]: This method is never used in the latest code

Comment (Sarkar) [inline Branch-SDK/src/io/branch/referral/SystemObserver.java]: makes sense to me

**Comment by Sarkar** [comment]
👍

**Pull request merged**